docs/articles/documentation/test-api/test-code-structure.md

```diff
@@ -572,3 +573,37 @@ test('Fixture2Test3', () => {});
 
 // Only tests in Fixture1 and the Fixture2Test2 test are run
 ```
+
+## Disable Page Caching
+
+The browser's local storage is not preserved between [role](authentication/user-roles.md) actions and the rest of the test code due to page caching. You need to disable page caching to share the local storage content.
+
+Use the `fixture.disablePageCaching` and `test.disablePageCaching` methods to disable caching during a particular fixture or test.
```

Comment:
The usage seems to be similar to `only` and `skip`:

```js
fixture
    .disablePageCaching `My fixture`
    .page `https://example.com`;

test
    .disablePageCaching
    ('My test', async t => { /* ... */ });
```

and 

```js
fixture.skip `Fixture1`;

test.skip('Fixture2Test2', () => {});
```

And we describe `only` and `skip` as methods. So, they are all properties?

Reply:
Yes, technically `skip` and `only` are properties too.

fixture.method()('test', async t => {}) - method
fixture.property('test', async t => {}) - property.

@kirovboris @AndreyBelym Should we describe `disablePageCaching` as method or property? Or something else?

`

Reply:
Mocha doesn't use any `method` or `property` labels.
https://mochajs.org/#exclusive-tests
In our case, it's method, because it can be called 
```js
test.disablePageCaching('testName');
```
In the last example it's method in both case, because it's a function

fixture.fn()('test', async t => {})
fixture.fn('test', async t => {})

Reply:
I describe the `property` or `method`  not in general sense.
I write about `method` or `property` of the chain.
See another example:
```js
expect(true).to.be.true;

expect - method
to, be, true - properties.
```

After fresh look, I guess we can use the `function` word in this case.

Reply:
Ok, @VasilyStrelyaev your turn

Reply:
To my mind, a property is something you assign to, or use the returned value. Here we do neither. Technically, it's a getter, as far as I can see, but we use a side effect to set the value, don't we? So, the use is more like a method.
As to 'function vs method', when you use a dot `fixture.disablePageCaching`, it looks like a method, doesn't it?

However, I may not be aware of the latest trends, so that's just how I'm used to saying things...